README.md

```diff
@@ -39,18 +39,12 @@ linker = "/Users/yourusername/opt/bin/x86_64-pc-elf-gcc"
 
 Where `yourusername` is your username.
 
-This project uses the xargo crate. To get it:
-
- Before install necessary dependencies, for Ubuntu 16.04, we need to install libssl-dev:
- ```bash
- $ sudo apt-get install libssl-dev
- ```
-
-Now install xargo:
+This project uses the xargo crate (https://github.com/japaric/xargo). To get it:
```

Comment:
Probably should link with markup here, e.g. `[xargo](https://github.com/japaric/xargo)`